app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -144,18 +144,8 @@ XrResult OpenXRInputSource::Initialize()
 
         if (mSupportsFBHandTrackingAim)
             mGestureManager = std::make_unique<OpenXRGestureManagerFBHandTrackingAim>();
-        else {
-            switch (deviceType) {
-                case device::MagicLeap2:
-                    // Disable filtering for ML2, data is already quite good.
-                    mGestureManager = std::make_unique<OpenXRGestureManagerHandJoints>(mHandJoints);
-                    break;
-                default:
-                    // TODO: fine tune params for different devices.
-                    OpenXRGestureManagerHandJoints::OneEuroFilterParams params = { 0.25, 0.1, 1 };
-                    mGestureManager = std::make_unique<OpenXRGestureManagerHandJoints>(mHandJoints, &params);
-            };
-        }
+        else
+            mGestureManager = std::make_unique<OpenXRGestureManagerHandJoints>(mHandJoints);
```

Comment:
Actually this is the only thing I think we should revert. The other part of the patch that adds the algorithm params code is very useful in case we want to customize the values for different devices

Reply:
That's true, although we don't have any param customization for now. In that regard, I think I'll just revert the change that disable the filter, and will work on customization for ML2 device in a follow-up patch, if you don't mind. I think it's more urgent to get back the usability we had with the filter.

Reply:
Sure, I was not asking for fine-tuning the ML2, just to avoid the removal of the parametrization.